cpp/src/arrow/util/logging.h

```diff
@@ -94,8 +94,7 @@ class NullLog {
 class CerrLog {
  public:
   CerrLog(int severity)  // NOLINT(runtime/explicit)
-      : severity_(severity),
-        has_logged_(false) {}
+      : severity_(severity), has_logged_(false) {}
```

Comment:
I upgraded to 4.0 recently. I'll look into keeping my older version or making sure that I run the 3.9 fixes if possible.